src/ui/public/agg_types/metrics/scripted_metric.js

```diff
@@ -0,0 +1,44 @@
+define(function (require) {
+
+  return function AggTypeMetricScriptedMetricProvider(Private) {
+    var _ = require('lodash');
+
+    // KBN 5
+    // var MetricAggType = Private(require('ui/agg_types/metrics/metric_agg_type'));
+
+    var MetricAggType = Private(require('ui/agg_types/metrics/MetricAggType'));
+    var fieldFormats = Private(require('ui/registry/field_formats'));
+    var stringEditor = require('ui/agg_types/controls/string.html');
+
+    return new MetricAggType({
```

Comment:
I used the example at https://www.elastic.co/guide/en/elasticsearch/reference/2.4/search-aggregations-metrics-scripted-metric-aggregation.html for testing this metric:

- the metric label is not set

    ![2016-10-24-170215_360x442_scrot](https://cloud.githubusercontent.com/assets/795879/19653452/85fb3344-9a0c-11e6-886e-4b92beb208cf.png)

    - this has also an impact when using `terms` bucket when you choose the terms' order

- some error when ordering the `terms` bucket with the scripted metric

    ![2016-10-24-170538_1610x36_scrot](https://cloud.githubusercontent.com/assets/795879/19653385/477c21d2-9a0c-11e6-858d-d6036f2392da.png)